src/views/GovernanceTaskList/index.test.tsx

```diff
@@ -187,23 +183,21 @@ describe('The Goveranance Task List', () => {
             </Provider>
           </MemoryRouter>
         );
-
-        const id = 'GovernanceTaskList-SecondaryList';
-        component
-          .find('button[data-testid="remaining-steps-btn"]')
-          .simulate('click');
-        setImmediate(() => {
-          component.update();
-          expect(
-            component.find(`button[aria-controls="${id}"]`).exists()
-          ).toEqual(true);
-          expect(component.find(`ol#${id}`).exists()).toEqual(true);
-          done();
-        });
       });
+
+      const id = 'GovernanceTaskList-SecondaryList';
+      component!
```

Comment:
Because of the way that `component` is declared and set above, TypeScript is seeing that it could be `undefined` here. Adding `!` removes the warning and will raise an error if that is ever the case.